makemake.sh

```diff
@@ -34,16 +34,26 @@ shopt -s nocasematch
 # for mode in AVX512 AVX2 AVX SSE2; do
 	# if grep -iq "$mode" /proc/cpuinfo; then
 		# echo -e "The CPU supports the ${mode} SIMD build mode.\n"
-		# ARGS+=( "-DUSE_${mode}" )
+		# CPP_ARGS+=( "-DUSE_${mode}" )
 		# break
 	# fi
 # done
 
+
 DIR=obj
 Mlucas=Mlucas
 Mfactor=Mfactor
 TARGET=$Mlucas
-ARGS=(-DUSE_THREADS) # Optional compile args
+
+# Optional compile [optimize] args
+C_ARGS=(-fdiagnostics-color -Wall)
+C_ARGS_OPT=(-g -O3)
+C_ARGS_DEBUG=(-g -Og)
+mapfile -t C_ARGS_ADD <<< "$XCFLAGS"
+C_ARGS+=("${C_ARGS_ADD[@]}")
```

Comment:
`XCFLAGS` is only a single line:
```suggestion
C_ARGS+=("$XCFLAGS")
```